fbw-a380x/src/systems/instruments/src/ANC/instrument.tsx

```diff
@@ -0,0 +1,141 @@
+import { EventBus, FSComponent, HEventPublisher } from '@microsoft/msfs-sdk';
+
+var flightphase = {
+  "WelcomeOnboard": 0,
+  "SafetyBriefing": 0,
+  "CrewSeatsTakeoff": 0,
+  "SeatbeltSignOff": 0,
+  "StartingDecent": 0,
+  "CrewPrepareLanding": 0,
+  "ArrivalDeboarding": 0
+};
+/**
+ * Resetting the flightphase to simulate turnaround or new flight
+ */
+
+
+
+
+/**
+ * 
+ * @param varCat - Category if either L Var or A var 
+ * @param variableName - Name of the SimVar to be executed
+ * @param type - Type of SimVar
+ * @param value - Value of the SimVar, for L Vars consider only Numbers
```

Comment:
Those - will make the jsdoc render oddly.
```suggestion
 * @param varCat Category if either L Var or A var 
 * @param variableName Name of the SimVar to be executed
 * @param type Type of SimVar
 * @param value Value of the SimVar, for L Vars consider only Numbers
```